modules/examples/todolist/runtime/src/main/scala/CommonRuntime.scala

```diff
@@ -16,10 +16,10 @@
 
 package examples.todolist.runtime
 
-import monix.execution.Scheduler
+import scala.concurrent.ExecutionContext
 
 trait CommonRuntime {
 
-  implicit val S: Scheduler = Scheduler.Implicits.global
+  implicit val EC: ExecutionContext = scala.concurrent.ExecutionContext.Implicits.global
```

Comment:
If you are using `import scala.concurrent.ExecutionContext` in line 19, this default value could be shorter `ExecutionContext.Implicits.global`. :wink: